init

```diff
@@ -208,6 +208,11 @@ function createSymlink($root, $links) {
         echo "      symlink " . $root . "/" . $target . " " . $root . "/" . $link . "\n";
         //first removing folders to avoid errors if the folder already exists
         @rmdir($root . "/" . $link);
+        //next removing existing symlink in order to update the target
+        $exist = @readlink($root . "/" . $link);
```

Comment:
Well, this will preserve modification time. Maybe someone will need this for some reason. :)